test/execute_noCLI.test.js

```diff
@@ -0,0 +1,195 @@
+                            "drift": "0",
+                            "geo_point": null,
+                            "instanceID": "f70831ae-863f-4a20-909f-e0ffacebd85e",
+                            "timeEnd": "2020-04-21T18:52:00.482000Z",
+                            "timeStart": "2020-04-21T18:51:06.456000Z",
+                            "userID": "cbc657317f74c7711d167f916afcf966",
+                            "username": "aleksa@openfn.org"
+                        },
+                        "patient_identifier_information": {
+                            "basic_demo": {
+                                "patient_family_name": "Msikinya",
+                                "patient_first_name": "Rachel"
+                            },
+                            "confirmed": {
+                                "confirm_label": "",
+                                "confirmed_label": ""
+                            },
+                            "covid_id": {
+                                "auto_uid": "BONRESR",
+                                "auto_uid_label": "",
+                                "manual_or_auto": "generate",
+                                "unique_case_id": "BONRESR"
+                            },
+                            "patient_case_status": "confirmed",
+                            "patient_name_last_first": "Msikinya, Rachel"
+                        }
+                    },
+                    "id": "f70831ae-863f-4a20-909f-e0ffacebd85e",
+                    "indexed_on": "2020-04-21T18:52:01.323139",
+                    "initial_processing_complete": true,
+                    "is_phone_submission": true,
+                    "metadata": {
+                        "appVersion": "Formplayer Version: 2.47",
+                        "app_build_version": null,
+                        "commcare_version": null,
+                        "deviceID": "Formplayer",
+                        "drift": "0",
+                        "geo_point": null,
+                        "instanceID": "f70831ae-863f-4a20-909f-e0ffacebd85e",
+                        "location": null,
+                        "timeEnd": "2020-04-21T18:52:00.482000Z",
+                        "timeStart": "2020-04-21T18:51:06.456000Z",
+                        "userID": "cbc657317f74c7711d167f916afcf966",
+                        "username": "aleksa@openfn.org"
+                    },
+                    "problem": null,
+                    "received_on": "2020-04-21T18:52:00.654283Z",
+                    "resource_uri": "",
+                    "server_modified_on": "2020-04-21T18:52:00.820344Z",
+                    "type": "data",
+                    "uiversion": "1",
+                    "version": "2",
+                    "configuration" : {
+                        "username" : "admin",
+                        "password" : "district",
+                        "hostUrl"  : "https://play.dhis2.org/2.34.1"
```

Comment:
If dhis2 is necessary, should this be mocked?

Reply:
It probably would be better to mock it, however I feel like you are right about keeping it without adaptors in the core.